src/otx/core/model/entity/action_classification.py

```diff
@@ -39,6 +60,7 @@ def __init__(self, num_classes: int, config: DictConfig) -> None:
         config = inplace_num_classes(cfg=config, num_classes=num_classes)
         self.config = config
         self.load_from = config.pop("load_from", None)
+        self.image_size = (1, 1, 3, 8, 224, 224)
```

Comment:
Why are there six dimensions? Is it `NCTHW`?

Reply:
Can you find a root cause of this and fix in the next PR?

Reply:
I'm not sure, mmaction do
![image](https://github.com/openvinotoolkit/training_extensions/assets/106788349/709c264d-b97e-468c-ada7-b66b2f7c30be)
during feature extraction
![image](https://github.com/openvinotoolkit/training_extensions/assets/106788349/6163bb99-5380-467e-bd1d-9970e86d5a8b)
It is used train, predict, export

Reply:
I can see that the default test pipeline uses `ThreeCrop`
https://github.com/open-mmlab/mmaction2/blob/4d6c93474730cad2f25e51109adcf96824efc7a3/configs/recognition/x3d/x3d_m_16x5x1_facebook-kinetics400-rgb.py#L18
If so, isn't it correct that it should be `(1,3,3,8,224,224)`?

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/106788349/b79a9560-5d31-48ab-ab43-6ed337a47d1d)
OTX does CenterCrop

Reply:
Did you evaluate on the side effect about that mismatch?

Reply:
I will during fixing performance issue in OTX2.0. Actually OTX1.x show good performance with previous pipelines